index.html

```diff
@@ -1,19 +1,85 @@
+        <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    </head>
+    <body>
+        <header class="header">
+            <img src="./img/karma-logo.svg" alt="Karma logo" class="logoImg"/>
+            <nav>
+                <ui class="navList">
+                    <li class="navItem">
+                        <a href="#" class="navLink"><span class="makeBlack">Meet Kamra</span></a>
+                    </li>
+                    <li class="navItem">
+                        <a href="#" class="navLink">How It works</a>
+                    </li>
+                    <li class="navItem">
+                        <a href="#" class="navLink">store</a>
+                    </li>
+                    <li class="navItem">
+                        <a href="#" class="navLink">Blog</a>
+                    </li>
+                    <li class="navItem">
+                        <a href="#" class="navLink">Help</a>
+                    </li>
+                    <li class="navItem">
+                        <a href="#" class="navLink">Login</a>
+                    </li>
+                </ui>
+            </nav>
+            <img class="hamburgerMenu" src="./img/menu-hamburger.svg" alt="menu bar"/>
+        </header>
+        <main>
+            <section class="introduction">
+                <h1>Introducing Karma</h1>
+                <p>Bring WiFi with you,everywhere you go.</p>
+                <button class="learnmoreBtn">learn more</button>
+            </section>
+            <section>
+                <div class="advantageTitle">
+                    <p>Everyone needs a little Karma.</p>
+                </div>
+                
+                <div class="advantage">
+                    <div class="advantageColumn">
+                        <img src="./img/icon-devices.svg" alt="devices picture">
+                        <p>Internet for all devices</p>
+                    </div>
+                    <div class="advantageColumn">
+                        <img src="./img/icon-coffee.svg" alt="coffee picture">
+                        <p>Boosts your productivity</p>
+                    </div>
+                    <div class="advantageColumn">
+                        <img src="./img/icon-refill.svg" alt="refill picture"> 
+                        <p>Pay as You Go</p> 
+                    </div>
+                </div>    
+            </section>
+            <section class="getKarma">
+                <div class="getkarmaBackground">
+                </div>
+                <div class="getkarmaTitle">
+                    <p><em>whenever I am,I just don't worry about my connection any more</em></p>
+                    <button class="learnmoreBtn">Get karma today</button>
+                </div>  
+            </section>
+        </main>
+        <footer>
+            <hr>
+            <h3>
+                join us on
+            </h3>
+            <a href="#"><img src="img/twitter-icon.svg" alt="Twitter logo"></a>  
```

Comment:
Is there a semantic tag that could be used to group these links?

Reply:
can I use <section> ? @Sam-Martin